politeiawww/api/www/v1/v1.go

```diff
@@ -285,6 +286,7 @@ const (
 	NotificationEmailAdminProposalVoteAuthorized EmailNotificationT = 1 << 6
 	NotificationEmailCommentOnMyProposal         EmailNotificationT = 1 << 7
 	NotificationEmailCommentOnMyComment          EmailNotificationT = 1 << 8
+	NotificationEmailProposerMessage             EmailNotificationT = 1 << 9
```

Comment:
I don't think we'd want to make this a notification setting because then the user would be able to turn it off.